README.md

```diff
@@ -38,10 +42,11 @@ Serverpod's future calls replace complicated cron jobs. Call a method anytime in
 Monitor the database and external services that you are using. Write custom health checks and get notified when something goes wrong.
 
 ### Easy deployment
-Serverpod comes with Terraform scripts for GCP and AWS which makes it very quick to deploy your server. We are still working on scripts for other platforms. Please get in touch with us if you want to [contribute](https://docs.serverpod.dev/contribute).
+Serverpod comes with Terraform scripts for Google Cloud Platform and AWS, making deploying your server very quick. We are still working on scripts for other platforms. Please get in touch with us if you want to [contribute](/contribute).
```

Comment:
This needs to be a full http link to our webpage since the file is in our GitHub repo.